prow/gerrit/client/client.go

```diff
@@ -404,6 +405,22 @@ func (c *Client) GetChange(instance, id string, addtionalFields ...string) (*Cha
 	return info, nil
 }
 
+func (c *Client) SubmitChange(instance, id string, wait bool) (*ChangeInfo, error) {
+	c.lock.RLock()
+	defer c.lock.RUnlock()
+	h, ok := c.handlers[instance]
+	if !ok {
+		return nil, fmt.Errorf("not activated gerrit instance: %s", instance)
+	}
+
+	info, resp, err := h.changeService.SubmitChange(id, &gerrit.SubmitInput{WaitForMerge: wait})
```

Comment:
What does `WaitForMerge` do? I don't see this field documented in the API spec or the go-github library: https://gerrit-review.googlesource.com/Documentation/rest-api-changes.html#submit-input
Does it cause the call to block until the merge is complete or does it mean something else?

Reply:
It's documented here https://pkg.go.dev/github.com/andygrunwald/go-gerrit#SubmitInput

Reply:
Seems like from an old api https://gerrit-documentation.storage.googleapis.com/Documentation/2.11/rest-api-changes.html#submit-change, this is probably deprecated according to https://javadoc.io/static/com.google.gerrit/gerrit-extension-api/3.4.0/deprecated-list.html

Reply:
go-gerrit still has it at HEAD though https://github.com/andygrunwald/go-gerrit/blob/41749a62809298f29d8bdf8b21d946853972ec44/changes.go#L207